tfe/resource_tfe_policy_set_test.go

```diff
@@ -465,3 +541,88 @@ resource "tfe_policy_set" "foobar" {
   global       = true
   policy_ids   = ["${tfe_sentinel_policy.foo.id}"]
 }`
+
+func testAccTFEPolicySet_vcs() string {
+	return fmt.Sprintf(`
+variable "github_token" {
+  default = "%s"
+}
+
+variable "vcs_identifier" {
+  default = "%s"
+}
+
+variable "vcs_branch" {
+  default = "%s"
+}
+
+variable "vcs_policies_path" {
+  default = "%s"
+}
```

Comment:
Let's not make the tests more complicated as they need to and just interpolate the values where we need them instead of adding variables. In these tests this doesn't add value other that making the test code longer.

Reply:
Not 100% true:

* At a general level, adding verbs as variables allows the verbs to be passed through `terraform fmt -` without risk of failures. This was more a TF 0.11 and earlier thing, but it's worth noting.
* If referenced more than once, it reduces stutter in the Sprintf generation.

I'll change this though as this is not necessarily the case here.